docs/environment-setup.md

```diff
@@ -88,7 +88,7 @@ sudo add-apt-repository \
    1. Create the Docker group: `groupadd docker`
    1. Add your user to the group: `sudo usermod -aG docker $USER`
    1. Log out and log back in (you may have to restart if you are running through a virtual machine)
-9. Verify your installation by running `docker run hello-world`. This should print a hello world paragraph that includes a confirmation that Docker is working on your system.
+9. Make sure to you have docker running in your machine, either by `sudo dockerd` or on startup. Verify your installation by running `docker run hello-world`. This should print a hello world paragraph that includes a confirmation that Docker is working on your system.
```

Comment:
running _on_ your machine, you can start docker through the following methods:
1. running the command `sudo dockerd`
2. starting the docker application manually
3. restarting your machine
Verify your installation by running...

Reply:
That's a good idea. Maybe have a separate step before "Verify your installation by running..." that explains the methods to run docker first

Reply:
By the way, if we restart our machine won't we have to restart dockerd and the GUI manually again?

Reply:
I don't think so, because the next time you restart, docker will be started up startup. I think that's what one of the commands do

Reply:
Care to experiment =)

Reply:
Yeah, something worth experimenting. I push the new changes you suggested. Thanks again for the suggestions!